src/components/apiDelegation/reusables/OverviewPageContent/OrgDelegationAccordion/OrgDelegationAccordion.module.css

```diff
@@ -1,6 +1,19 @@
-.accordionContent{
+@media only screen and (max-width: 768px) { 
+  .accordionContent{
+    padding-left: 0rem;
```

Comment:
Without the padding, the content looks a bit like its not really inside the accordion's dropdown.
Maybe add enough padding to align it with the middle of the arrow? (1rem)

![image](https://user-images.githubusercontent.com/22012997/224708987-e4d04ba1-cb8a-4b2c-ba23-6bb1c6453dcf.png)

To save some space we can probably scale down the margin-right of .ListItemTexts in the deletable list items.
A margin of 0.7rem is probably enough.
Together, the changes will look like this:
![image](https://user-images.githubusercontent.com/22012997/224710680-ac4b8651-6fa0-46d8-af38-7eb63de34457.png)